packages/help-extension/src/index.tsx

```diff
@@ -232,6 +232,8 @@ const open: JupyterFrontEndPlugin<void> = {
       content.addClass(HELP_CLASS);
       content.title.label = text;
       content.id = `${namespace}-${++counter}`;
+      //sets loading to lazy
+      content.node.querySelector('iframe')?.setAttribute('loading', 'lazy');
```

Comment:
It would be good to move this implementation detail into `IFrame` implementation, this is (a) add an option for `loading` (b) pass this option into the constructor here in line 229 and in the other file where it is used:

https://github.com/jupyterlab/jupyterlab/blob/b2ff3fbbe60e1d08d23f6a4738d4f2d76f175152/packages/ui-components/src/components/iframe.ts#L9-L13

Reply:
Hi @krassowski if you're able to check out the changes I made. I added your suggestion of having an option for loading in the constructor of iFrame and updated the html and help rendering widgets to have the loading option set to lazy. Thanks!